packages/cli/src/commands/generate/helpers.js

```diff
@@ -170,3 +171,22 @@ export const intForeignKeysForModel = (model) => {
     .filter((f) => f.name.match(/Id$/) && f.type === 'Int')
     .map((f) => f.name)
 }
+
+export const isWordNonPluralizable = (word) => {
+  return pluralize.isPlural(word) === pluralize.isSingular(word)
+}
+
+/**
+ * @description Adds an s, if it it can't pluralize the word
```

Comment:
```suggestion
 * Adds an s if it can't pluralize the word
```

> If you describe a symbol at the very beginning of a JSDoc comment, before using any block tags, you may omit the @description tag.

https://jsdoc.app/tags-description.html